onnxscript/version_converter/_version_converter.py

```diff
@@ -19,6 +19,10 @@
 CURRENT_MAX_ONNX_OPSET = 23
 
 
+class VersionConverterError(Exception):
```

Comment:
Prefer a RuntimeError. It is easier to catch

```suggestion
class VersionConverterError(RuntimeError):
```